### ISSUE TITLE
Journal Metrics API Error in 3.2.2

### ISSUE THREAD

**Issue by NickMeves** [Graylog2/graylog2-server#7592]
Upon upgrading from 3.1.4 to 3.2.2, the `api/cluster/<node-id>/journal` api endpoint is returning 400. (For all nodes, master or coordinating)

![image](https://user-images.githubusercontent.com/4123100/75578413-a7d9b880-5a18-11ea-95af-de9384f7716e.png)

We've reproduced this in 2 dev environments. This occurs for human admins (who login via SSO) and the default admin account (with SSO disabled).

Here is the API response JSON:

`{"type":"ApiError","message":"Unable to map property native_lib_dir.\nKnown properties include: flush_interval, plugin_dir, segment_age, max_age, bin_dir, data_dir, max_size, directory, flush_age, segment_size"}`

## Your Environment

3 Graylog nodes on Kubernetes with an OIDC SSO sidecar container in each pod. Backend still listening on 0.0.0.0:9000, but network policies in place so only Graylog pods can communicate to port 9000.

Master node detected in docker_entrypoint.sh looking at the POD_NAME environment variable passed in with the k8s downward API (so <pod_name>-0 is the master)variables

* Graylog Version: 3.2.2 (graylog/graylog:3.2.2-1)
* Elasticsearch Version: 6.8.6 (docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.6)
* MongoDB Version: 3.6.17 (mongo:3.6.17)
* Operating System: Kubernetes 1.13.12 on CoreOS
* Browser version: Chrome 79.0.3945.130

**Comment by NickMeves** [comment]
Under the hood -- journal still looks fine: index & log file are still being written to

```$ cd journal    
$ ls
graylog2-committed-read-offset	messagejournal-0  node-id  recovery-point-offset-checkpoint
$ cd messagejournal-0
$ ls
00000000000000196235.index  00000000000000196235.log
$ ls -l
total 136
-rw-r--r--. 1 graylog graylog 1048576 Feb 28 19:13 00000000000000196235.index
-rw-r--r--. 1 graylog graylog  128395 Feb 28 19:14 00000000000000196235.log```

**Comment by NickMeves** [comment]
Node journal metrics are still available individually:

![image](https://user-images.githubusercontent.com/4123100/75580770-2f292b00-5a1d-11ea-92ed-315e7128a9bd.png)

**Comment by bernd** [comment]
@NickMeves This is a duplicate of https://github.com/Graylog2/graylog2-server/issues/7521. The issue is already fixed and will be shipped in the upcoming 3.2.3 release. Thank you for the report!
